content/integrations/beanstalk.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Datadog-Elastic Beanstalk Integration
+integration_title: AWS Elastic Beanstalk
+kind: integration
+doclevel: basic
+---
+
+AWS Elastic Beanstalk is an easy-to-use service for deploying and scaling web applications and services developed with Java, .NET, PHP, Node.js, Python, Ruby, Go, and Docker on familiar servers such as Apache, Nginx, Passenger, and IIS.
+
+### Monitor Elastic Beanstalk environments with the Datadog agent container
+If you use Docker containers in a Beanstalk environment, and want to monitor you Docker usage, the containerized Datadog agent is the perfect tool!
```

Comment:
not sure about the `perfect tool!`, what about something that say we support it?